### PR TITLE
[INLONG-6848][DataProxy] Adapt original InlongMsg protocol and headers

### LINKED ISSUE #6848: [Improve][DataProxy] Support original InlongMsg protocol and headers
### Description

The mew sink layer sends PB msg instead of InlongMsg by default.

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
- Fixes #6848

### Modifications
 - data is converted to pb by default, support original default protocol and headers under the new sink layer
 - fix script config file name error

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
@luchunliang PTAL, thanks.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/tube/TubeHandler.java [hunk 1]
```diff
@@ -239,12 +240,16 @@ public void onException(Throwable ex) {
     private void sendSimpleProfileV0(SimpleBatchPackProfileV0 event, IdTopicConfig idConfig,
             String topic) throws Exception {
         // headers
-        Map<String, String> headers = event.getSimpleProfile().getHeaders();
+        Map<String, String> headers = event.getProperties();
+        if (MapUtils.isEmpty(event.getProperties())) {
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/tube/TubeHandler.java:244]:
```suggestion
        if (MapUtils.isEmpty(headers)) {
```

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/tube/TubeHandler.java [hunk 2]
```diff
@@ -239,12 +240,16 @@ public void onException(Throwable ex) {
...
+            headers = event.getSimpleProfile().getHeaders();
+        }
         // compress
         byte[] bodyBytes = event.getSimpleProfile().getBody();
         // sendAsync
         Message message = new Message(topic, bodyBytes);
         // add headers
+
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/tube/TubeHandler.java:252]: A strange blank line.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/kafka/KafkaHandler.java [hunk 3]
```diff
@@ -204,14 +205,18 @@ public void onCompletion(RecordMetadata arg0, Exception ex) {
...
+            headers = event.getSimpleProfile().getHeaders();
+        }
+        // body
         byte[] bodyBytes = event.getSimpleProfile().getBody();
         // sendAsync
         long sendTime = System.currentTimeMillis();
 
         // prepare ProducerRecord
         ProducerRecord<String, byte[]> producerRecord = new ProducerRecord<>(topic, bodyBytes);
+
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/kafka/KafkaHandler.java:219]: ditto.

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**